dataland-frontend/tests/component/testUtils/setup.ts

```diff
@@ -0,0 +1,89 @@
+import { routes } from "@/router";
+import { createPinia } from "pinia";
+import PrimeVue from "primevue/config";
+import DialogService from "primevue/dialogservice";
+import { createMemoryHistory, createRouter, Router } from "vue-router";
+import { mount } from "cypress/vue";
+import { VueWrapper } from "@vue/test-utils";
+import { DefineComponent } from "vue";
+import Keycloak from "keycloak-js";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
There are a lot of eslint-disable in this file. Is this still work in progress? If not we should discuss if there is no way to avoid them.

Reply:
This is intended to enforce that for lksg datasets, the reporting period equals the year of the dataDate.
For other frameworks the `getRandomReportingPeriod()` decides the value of reportingPeriod.

But I agree that this is confusing on first sight.
I will check now if I can enforce this in a better readable way.